java/src/jmri/jmrix/loconet/LnReporter.java

```diff
@@ -198,7 +229,6 @@ public void setState(int s) {
       */
     @Override
     public void dispose() {
-        tc.removeLocoNetListener(~0, this);
         super.dispose();
     }
```

Comment:
This method is now redundant.  It could be removed, allowing the parent object to perform its disposal "behind the scenes".